data/moves.js

```diff
@@ -5319,7 +5319,7 @@ let BattleMovedex = {
 			}
 			for (const ally of target.side.active) {
 				if (ally && this.isAdjacent(target, ally)) {
-					this.damage(ally.maxhp / 16, ally, source, move);
+					this.damage(ally.maxhp / 16, ally, source, 'flameburst');
```

Comment:
```suggestion
					this.damage(ally.maxhp / 16, ally, source, Dex.getEffect('Flame Burst'));
```

There's no need for modifying `.damage`.

Although I'm still more in favor of just splitting up `moveDamage` and `damage`.

Reply:
No, I intentionally whitelisted generic effects. I just don't want effects actually named after a move to be in the `battle.damage` whitelist.